source/conf.py

```diff
@@ -24,9 +24,9 @@
 author = 'Apache CloudStack Project'
```

Comment:
Change (above this line) copyright to 2020?

Reply:
Fixed.